### PR TITLE
Blank slate

### PR DESCRIPTION
**Pull request by gjjones**

### CODE REVIEW

**Comment by gjjones** [comment]
This is not to say none of that stuff is useful. We will still be bringing over the BEM/atomic bits and other goodies as needed. The intent is to give us a clear slate to build back up.

**Comment by wylie** [comment]
Could you also remove all of the GruntIcon stuff? I still think that all of that should become a component in Buzz React. Perhaps that is too much for this _blank slate_ PR?

File: less/_mixins.less [hunk 1]
```diff
@@ -48,18 +39,23 @@
 
 // hidden (accessible by screen readers)
 .hidden {
```
Comment (wylie) [inline less/_mixins.less:41]: this and `.off-page` really need fixing… just saying

File: less/_mixins.less [hunk 2]
```diff
@@ -138,137 +134,6 @@
 }
 
 
-//----- FIXED WIDTH COLUMNS -----------------------------//
-// takes two arguments, a width and a side
-// w should be any number, minus the units
-// s should be lowercase R or L
-.fixedCol(@w, @s) {
```
Comment (wylie) [inline less/_mixins.less:326]: So happy to see this go

**Comment by esimons** [comment]
@wylie we're assuming IE9+ with the React stuff now, so Grunticon I don't think needs to even be in the picture.

**Comment by gjjones** [comment]
@esimons, Some conversation is getting lost with some of it happening in flowdock instead. There's a task for SVGs to be dealt with on this US. It'll be dealt with once this settles. [This one](https://css-tricks.com/creating-svg-icon-system-react/) is the current inspiration for a new direction.

File: less/_mixins.less (hunk 1 above)
Comment (gjjones) [inline less/_mixins.less:41]: I updated the contents of that one to reflect what is in the H5BP. Safe to assume their snippet is pretty well tested

Comment (gjjones) [inline less/_mixins.less:41]: Not `.off-page` though. That one definitely needs revisiting. Might not even be needed

**Comment by wylie** [comment]
@esimons, sure, that's awesome. I don't care whether or not we use GruntIcon. I'm just of the mind that SVG's should come out of Edna

File: less/_mixins.less (hunk 1 above)
Comment (wylie) [inline less/_mixins.less:41]: Right, saw that and it makes me happy. I think the names should also be changed to reflect more of what they do. Hidden from everything vs hidden from a screen reader, vs hidden from desktop but not a screenreader vs Godzilla vs whatever…

**Comment by wylie** [comment]
@gjjones I'm all good with this PR, but I'm sure I can continue to comment on it…

**Comment by esimons** [comment]
@wylie @gjjones Gotcha, I misunderstood. 100% on board with components encapsulating icons instead of baking into Edna.

**Comment by wylie** [comment]
That's right you misunderstood! To the Battle Cube!

**Comment by esimons** [comment]
soundthehornofbattle

**Pull request merged**